### ISSUE TITLE
great work! could you provide some training advice about iteration number and lr

### ISSUE THREAD

**Issue by LongWayGuo** [facebookresearch/vggt-omega#30]
first, in my experience,  for [2,16] , max 24 image , it would taken 5s/iter for A100.  so for 240K iter as you mentioned in the implementation detail it would take about 13.8day for training , is that correct? I think it's too long for one time training experiment. do you have some advice for it?

and second. in the section of "How to fine-tune vggt/vggt-Ω" , you mentioned a warm-up followed by cosine decay. and keep the peak lr small. so how small it should be?  in my experience for a specific scenario(like Autonomous Driving with our internal dataset), I set 5% warm-up with peak lr 5e-5 then cosine decay. but I found after before the lr reach to peak. the model's performance already degenerate compared with the original edition.  especially for the pose output(because we observe that most time the depth in local camera is correct but the points in the reference camera are misaligned across frames)

but after a full training round(likes 500iter*60epoch*40A100 with 4day, opt all parameter except DINO),   we could get a better result then original. which I think we have got the data domain transfer. but we want to accelerate the fine-tuning process.  as we think a domain shift should need too long , and we can't explain for a fine-tune process with warm-up,  why the model performance would degrade first then converge to a better.  is there any config could make sure our fine-tune process always step better and better. and what's the best parameter for fine-tune?

thank you for your attention :)

**Comment by jytime** [comment]
Hi,

1. For fine-tuning, I think 240K is not necessary. Usually 20k/40k or fewer are good enough.
2. Yeah that's exactly correct and consistent with our own observations. I would usually suggest finetuning with a peak lr like 2e-5 or 1e-5. And even using such small lr, you can see a clear performance drop around the period of the peak lr. It is expected. You will see the performance become better and become after finishing 70% of the cycle. 
3. Generally, I would not hope the "fine-tune process always step better and better", becuase I found, the performance drop in the early stage is usually beneficial in the end(probably something related to forgetting proir). But if you really care about the intermediate performance during training, you can try to use a smaller lr, and skip the warm up process (personally i would not suggest so).
